### ISSUE TITLE
[polyfill] limitations/caveats

### ISSUE THREAD

**Issue by mayank99** [mayank99/open-styleable#6]
- client script doesn't watch for changes to stylesheets
- nesting declarative and imperative shadow roots does not work perfectly, because the DSD transform is only executed once (way before the client script runs)
- implicit `@scope` may cause issues (see #4)

**Comment by bkardell** [comment]
Feels like this should be (at least currently) in the readme

**Comment by mayank99** [comment]
good point! i've linked this issue in the readme https://github.com/mayank99/open-styleable/commit/096d1d1c68b48b9f779720efa357caa2c44f93c3
